cmd/reviewdog/main.go

```diff
@@ -28,6 +28,8 @@ import (
 	"github.com/mattn/go-shellwords"
 )
 
+var Export = 1
```

Comment:
**[unused]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
var Export is unused (U1000)